utils/vlog/tests/dsp_out_registered/golden.pb_type.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0"?>
```

Comment:
We might want to add the `<?xml version="1.0"?>` header back...

Reply:
Ok, I missed that